CHANGELOG.md

```diff
@@ -168,6 +168,9 @@ Buffers for state serialization instead of Amino.
 * (server) [\#5709](https://github.com/cosmos/cosmos-sdk/pull/5709) There are two new flags for pruning, `--pruning-keep-every` 
 and `--pruning-snapshot-every` as an alternative to `--pruning`. They allow to fine tune the strategy for pruning the state.
 * (crypto/keys) [\#5739](https://github.com/cosmos/cosmos-sdk/pull/5739) Print an error message if the password input failed.
+* (client) [\#5810](https://github.com/cosmos/cosmos-sdk/pull/5810) Added new `--offline` flag improving the user experience. This
```

Comment:
WDYT about:

```
Added a new `--offline` flag that allows commands to be executed without an internet connection.
Previously, `--generate-only` served this purpose in addition to only allowing txs to be generated.
Now, `--generate-only` solely allows txs to be generated without being broadcasted and disallows
Keybase use and `--offline` allows the use of Keybase but does not allow any functionality that
requires an online connection.
```

Reply:
I see what you mention, but the part that does not access to the Keybase I applied when offline is provided. 

https://github.com/cosmos/cosmos-sdk/blob/0621e5778303a2e5876e052576b78c96308d3ea2/client/context/context.go#L283

This should belong to generate only then?